distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/Client.java

```diff
@@ -142,23 +132,15 @@ public static void main(String args[]) {
             LOGGER.info("Unable to obtain IOR File :  Must specify URL to IOR file.");
         }
         try {
-            client.runTest(args);
+            client.runTest(args[0]);
```

Comment:
Should we have a JavaDoc comment on `main` or on the class that explains what the expected argument is?